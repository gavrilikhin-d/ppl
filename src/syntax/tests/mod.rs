use crate::test_compilation_result;

test_compilation_result!(consume_greater);
test_compilation_result!(escaped_id);
test_compilation_result!(multiple_errors);
