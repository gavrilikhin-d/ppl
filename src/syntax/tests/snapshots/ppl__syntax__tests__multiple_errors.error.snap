---
source: src/syntax/tests/mod.rs
expression: err
---
Error: lexer::unexpected_token

  × expected `=`, got `integer`
   ╭─[multiple_errors.ppl:1:7]
 1 │ let x 1
   ·       ─
 2 │ if
   ╰────
parser::missing_expression

  × missing expression
   ╭─[multiple_errors.ppl:2:3]
 1 │ let x 1
 2 │ if
   ╰────
